modulus/distributed/manager.py

```diff
@@ -48,11 +48,13 @@ class ModulusUninitializedDistributedManagerWarning(Warning):
     """Warning to indicate usage of an uninitialized DistributedManager"""
 
     def __init__(self):
-        message = "A DistributedManager object is being instantiated before "
-        "this singleton class has been initialized. Instantiating a manager before "
-        "initialization can lead to unexpected results where processes fail "
-        "to communicate. Initialize the distributed manager via "
-        "DistributedManager.initialize() before instantiating."
+        message = (
```

Comment:
Why are these changes needed?

Reply:
It wasn't printing the whole error message as shown in the comment here: https://github.com/NVIDIA/modulus/pull/549#issuecomment-2163600705